LondonBritishLibrary/orient/BLorient739.xml

```diff
@@ -53,52 +352,86 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">198</measure>
-                                    <measure unit="blank">2<note>at the beginning </note>
-                           </measure>
-                                    <measure unit="blank">2<note>at the end</note>
-                           </measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>371</height>
-                                        <width>324</width>
+                                    <measure unit="leaf" type="blank">4</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>14.625</height>
+                                        <width>12.75</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+                                <foliation>Two unnumbered blank leaves at the beginning, and two unnumbered leaves
+                                at the end.</foliation>
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns="3" writtenLines="29 30" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is written in a fine hand, of the earlier part of the <date>18th century</date>.
                                 </desc>
-                                
-                                <date notBefore="1700" notAfter="1800" resp="PRS10171Wright"/>
-                                
+                                <seg type="ink">Black, red.</seg>
+
+                                <date notBefore="1700" notAfter="1750" resp="PRS10171Wright"/>
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
+                          <list>
+                            <item xml:id="a1">
+                            <locus target="#1r"/>
+                            <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                            <q xml:lang="gez">አፈ፡ ወርቅ፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+                            </q>
+                          </item>
+                            <item xml:id="e1">
```

Comment:
Regardless of the addition type discussion, I would say that this is not an extra, but an addition